adcc/LazyMp.py

```diff
@@ -96,7 +96,8 @@ def t2(self, space):
     def td2(self, space):
         """Return the T^D_2 term"""
         if space != b.oovv:
-            raise NotImplementedError()
+            raise NotImplementedError("T^D_2 term not implemented "
+                                      f"for space {space}.")
```

Comment:
I'm pretty sure we can lift this now that we don't rely on libadc any more.

Reply:
Yes, but then I have to implement the amplitudes for CVS ☹️
I can give it a try...

Reply:
Don't bother if it's too annoying. It would just be nice to be able to get an MP(3) energy also for CVS-ADC(3).

Reply:
It's really annoying 😄 Let's do this in a follow-up, because it's taking too much time right now.